### PR TITLE
string-6

### PR DESCRIPTION
**Pull request by nguyenduykhanh1025**

### CODE REVIEW

File: src/main/java/String6.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+public class String6 {
+
+    public String getCovert(String str) {
+        StringBuffer strResult = new StringBuffer("");
+        for (int i = 0; i < str.length(); ++i) {
+            if (str.charAt(i) < 91) {
```
Comment (voquanghoa) [inline src/main/java/String6.java]: Charater.isLowerCase

File: src/main/java/String6.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                strResult.append((char) (str.charAt(i) + 32));
```
Comment (voquanghoa) [inline src/main/java/String6.java]: Dùng Character.toUpperCase

File: src/main/java/String6.java (hunk 1 above)
Comment (nguyenduykhanh1025) [reply src/main/java/String6.java]: done

File: src/main/java/String6.java (hunk 2 above)
Comment (nguyenduykhanh1025) [reply src/main/java/String6.java]: done

**Pull request merged**